src/main/java/core/basesyntax/strategy/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Action;
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String SEPARATION_PLACE = ",";
+    private static final int ACTION_PART = 0;
+    private static final int FRUIT_PART = 0;
+    private static final int QUANTITY_PART = 0;
+
+    public List<FruitTransaction> convertToTransactions(List<String> dateList) {
```

Comment:
The method name `convertToTransactions` seems to have a typo. It should probably be `dataList` instead of `dateList` as it is referring to a list of data.